server/routes/ad.ts

```diff
@@ -55,6 +55,8 @@ export default function (apiRouter: Router) {
   apiRouter.post('/detectors/{detectorId}/preview', previewDetector);
   apiRouter.get('/detectors/{detectorId}/results', getAnomalyResults);
   apiRouter.delete('/detectors/{detectorId}', deleteDetector);
+  apiRouter.post('/detectors/{detectorId}/_start', startDetector);
```

Comment:
np: I would not recommend to add `_` let's be consistent and remove it from kibana api.

Reply:
Sure, will change it